lab_1_classify_by_unigrams/main.py

```diff
@@ -1,68 +1,104 @@
 """
 Lab 1
-Language detection
+Language detection turi ip ip
 """
 
 
 def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
-
+    if type(text) is str:
```

Comment:
revert the doclines

Reply:
want to get type - use isinstace

Reply:
Try early return - already sad on a seminar

Reply:
done